### ISSUE TITLE
The role of self.weight in PaCoLoss

### ISSUE THREAD

**Issue by cxycxyzzz** [dvlab-research/Parametric-Contrastive-Learning#26]
Thank you for your work!

One thing bothers me is that when computing anchor_dot_contrast in line 44 of losses.py, self.weight is added with sup_logit. I wonder why this computation.

Thank you!

**Comment by jiequancui** [comment]
Thanks for your interest in our work.
Please refer to Long-tail learning via logit adjustment and Balanced Meta-Softmax for Long-Tailed Visual Recognition.
We integrate the center rebalance into the PaCo Loss.
